### ISSUE TITLE
get_historical data doesn't return correct results if there is only one row returned

### ISSUE THREAD

**Issue by tarlabs** [lukaszbanasiak/yahoo-finance#7]
If between start date and end date there is only one result row then data is not returned correctly. Below patch to the function fixes is it all

```
    def get_historical(self, start_date, end_date):
        """
        Get Yahoo Finance Stock historical prices

        :param start_date: string date in format '2009-09-11'
        :param end_date: string date in format '2009-09-11'
        :return: list
        """
        hist = []
        for s, e in get_date_range(start_date, end_date):
            try:
                query = self._prepare_query(table='historicaldata', startDate=s, endDate=e)
                result = self._request(query)
                if isinstance(result, dict):
                    result = [result]
                hist.extend(result)
            except AttributeError:
                pass
        return hist
```

**Comment by lukaszbanasiak** [comment]
@tarlabs Are you interested in create a pull request?

**Comment by tarlabs** [comment]
Okie, I will do that in 2-3 days time. Occupied with something else right now

**Comment by lukaszbanasiak** [comment]
Fixed in PR #12
